.github/workflows/build.yml

```diff
@@ -0,0 +1,121 @@
+name: build
+
+on: ['push']
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          path: '.'
+          pattern: '*.sh'
+          exclude: './.git/*'
```

Comment:
we should also exclude the `vendor` folder which is not in our control.

Reply:
Added 👍